lib/webauthn/authenticator_response.rb

```diff
@@ -10,8 +10,17 @@ def initialize(client_data_json:)
 
     attr_reader :client_data_json
 
+    def valid_type?
+      client_data.type == type
```

Comment:
One thing that helps me understand quickly when reading (and searching/grep'ing) source code why there's a call to a method that doesn't appear in the rest of the file is the following:

```ruby
def type
  raise NotImplementedError, "Please define #type method in subclass"
end
```

What do you think?

Reply:
yeah, that's a good habit actually... should have done it 👍